server/src/pages/shot/view.js

```diff
@@ -180,7 +180,7 @@ class Body extends React.Component {
 
   onClickDelete(e) {
     sendEvent("start-delete", "navbar", {useBeacon: true});
-    if (window.confirm("Are you sure you want to delete this shot permanently?")) {
+    if (window.confirm("Are you sure you want to delete this shot permanently?")) { // todo l10n: shotPageConfirmDeletion
```

Comment:
I think it would be helpful to have an imperative API just for these uses-cases. I imagine something like `formatString`. I'm not sure where to put it, however. The translations are managed by the `LocalizationProvider` component and are made available to `Localized` components via React's context.


What would be the best way to expose a method for retrieving translations from `LocalizationProvider`? Perhaps a HOC which connects a given component with the `LocalizationProvider` and passes a prop: the `formatString` method.